src/ert/run_models/multiple_data_assimilation.py

```diff
@@ -146,15 +123,19 @@ def run_experiment(
             self.run_workflows(
                 fixtures=PreExperimentFixtures(random_seed=self.random_seed),
             )
-            sim_args = {"weights": self._relative_weights}
+
+            # used to be self._relative_weights, which is "unscaled"
+            # is the weights before making their sum of reciprocals 1
```

Comment:
Seem to remember there was a reason for this 🤔 Were we showing it in the gui perhaps? Comment should be removed though, I must admit I am not reading commit by commit at the moment, but might make sense as its own commit (apologies if that is already the case)